src/components/Page.js

```diff
@@ -1,22 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import Sidebar from './Sidebar';
+import MobileMenuBar from './MobileMenuBar';
+import { mediaQueries } from './Common';
 
 const Container = styled.div`
   display: flex;
   align-items: stretch;
   min-height: 100vh;
 `;
 
+const LeftColumn = styled.div`
```

Comment:
How come this is called _Left_ column?

Reply:
omg it was 2am... the column is on the right. good review.

Reply:
Hm I was more referring to the fact that this container doesn't actually determine which side the content / sidebar is on